src/llmq/quorums_signing_shares.cpp

```diff
@@ -1125,7 +1127,12 @@ void CSigSharesManager::WorkThreadMain()
         didWork |= quorumSigningManager->ProcessPendingRecoveredSigs(*g_connman);
         didWork |= ProcessPendingSigShares(*g_connman);
         didWork |= SignPendingSigShares();
-        SendMessages();
+
+        if (GetTimeMillis() - lastSendTime > 100) {
+            lastSendTime = GetTimeMillis();
```

Comment:
nit: technically sending can take some time, so this should probably be bumped _after_ `SendMessages()`

Reply:
Agree and fixed